src/www/style.css

```diff
@@ -14,6 +14,14 @@
  * limitations under the License.
  */
 
+ body {
+  /*
+    we're not properly CSS resetting - this is to prevent
```

Comment:
Even though old comments are not doing so, comments should use proper capitalization and punctuation: https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar

Reply:
Ah, sorry. Fixed.